frontend/app/src/components/history/transactions/TransactionDetail.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <card>
+    <template #title>
+      {{ $t('transaction_details.title') }}
+    </template>
+    <div class="pt-4">
+      <v-row class="pb-2">
+        <v-col cols="2" class="font-weight-medium">
+          {{ $t('transaction_details.gas') }}
+        </v-col>
+        <v-col cols="10">
+          <amount-display :value="transaction.gas" integer />
+        </v-col>
+      </v-row>
+      <v-divider class="pb-2" />
+      <v-row class="pb-2">
+        <v-col cols="2" class="font-weight-medium">
+          {{ $t('transaction_details.gas_used') }}
+        </v-col>
+        <v-col cols="10">
+          <amount-display :value="transaction.gasUsed" integer />
+        </v-col>
+      </v-row>
+      <v-divider class="pb-2" />
+      <v-row class="pb-2">
+        <v-col cols="2" class="font-weight-medium">
+          {{ $t('transaction_details.gas_price') }}
+        </v-col>
+        <v-col cols="10">
+          <amount-display :value="toGwei(transaction.gasPrice)" asset="Gwei" />
+        </v-col>
+      </v-row>
+      <v-divider class="pb-2" />
+      <v-row class="pb-2">
+        <v-col cols="2" class="font-weight-medium">
+          {{ $t('transaction_details.nonce') }}
+        </v-col>
+        <v-col cols="10">{{ transaction.nonce }}</v-col>
+      </v-row>
+      <v-divider class="pb-2" />
+      <v-row class="pb-2">
+        <v-col cols="2" class="font-weight-medium">
+          {{ $t('transaction_details.input_data') }}
+        </v-col>
+        <v-col cols="10">
+          <textarea
+            :value="transaction.inputData"
+            readonly
+            spellcheck="false"
+            class="transaction-details__data text--secondary"
+          />
+        </v-col>
+      </v-row>
+    </div>
+  </card>
+</template>
+<script lang="ts">
+import { BigNumber } from '@rotki/common';
+import { defineComponent, PropType } from '@vue/composition-api';
+import { EthTransactionEntry } from '@/store/history/types';
+import { toUnit, Unit } from '@/utils/calculation';
+
+export default defineComponent({
+  name: 'TransactionDetail',
+  props: {
+    transaction: {
+      required: true,
+      type: Object as PropType<EthTransactionEntry>
+    }
+  },
+  setup() {
+    const toGwei = (value: BigNumber) => {
+      return toUnit(value, Unit.GWEI);
+    };
+
+    return {
+      toGwei
+    };
+  }
+});
+</script>
+<style scoped lang="scss">
```

Comment:
Since this is a new component please use css modules instead

Reply:
I think just keep using css scoped here?
_